### PR TITLE
Clarify limited write access in BigQuery docs

### PR DESCRIPTION
**Pull request by colebow**
## Description

Updating docs to remove inaccurate info

> Is this change a fix, improvement, new feature, refactoring, or other?

Improvement

> Is this a change to the core query engine, a connector, client library, or the SPI interfaces? (be specific)

Docs only

## Documentation

( ) No documentation is needed.
(x) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

(x) No release notes entries required.
( ) Release notes entries required with the following suggested text:

### CODE REVIEW

File: docs/src/main/sphinx/connector/bigquery.rst [hunk 1]
```diff
@@ -207,10 +207,10 @@ Retrieve all records stored in the partition ``_PARTITIONDATE = '2022-04-07'``::
 SQL support
 -----------
 
-The connector provides read and write access to data and metadata in
```
Comment (martint) [inline docs/src/main/sphinx/connector/bigquery.rst:210]: "create" and "drop" table qualify as write access. How is that inaccurate?

Comment (colebow) [reply docs/src/main/sphinx/connector/bigquery.rst:210]: I can clarify further - the intent behind this change is that you cannot actually write new data of any kind, which gets a little lost when you say "it provides write access to data."

**Review by hashhar — COMMENTED** [review]
We should be consistent and do what we have done in other connectors. i.e. mention both read and write in the paragraph and enumerate the specific ones that are supported in the bullet list.

e.g. https://github.com/trinodb/trino/blob/master/docs/src/main/sphinx/connector/sqlserver.rst#sql-support

File: docs/src/main/sphinx/connector/bigquery.rst [hunk 2]
```diff
@@ -207,10 +207,10 @@ Retrieve all records stored in the partition ``_PARTITIONDATE = '2022-04-07'``::
...
-the BigQuery database. In addition to the :ref:`globally available
-<sql-globally-available>` and :ref:`read operation <sql-read-operations>`
-statements, the connector supports the following features:
+The connector provides read access to data and metadata in the BigQuery
+database. In addition to the :ref:`globally available <sql-globally-available>`
+and :ref:`read operation <sql-read-operations>` statements, the connector
+supports the following features:
 
 * :doc:`/sql/create-table`
 * :doc:`/sql/drop-table`
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/bigquery.rst:216]: I would leave the read and write as it was ... but I am also confused .. are we saying this connector can create a table and drop a table but NOT do anything else with the table .. like no INSERT or whatever .. if thats the case I question the logic of adding create table support at all .. and would add support for INSERT and others as soon as possibiel.

**Comment by colebow** [comment]
Would it reasonable to include a "but write access is limited" or something like that to emphasize that you're only able to create tables/schemas?

**Review by ebyhr — APPROVED** [review]
Please fix commit message. The latest change is not "Remove inaccurate mention of write access"

**Pull request merged**